pkg/cluster/operation/destroy.go

```diff
@@ -111,7 +111,7 @@ func StopAndDestroyInstance(ctx context.Context, cluster spec.Topology, instance
 		// monitoredOptions for dm cluster is nil
 		monitoredOptions := cluster.GetMonitoredOptions()
 
-		if monitoredOptions != nil {
+		if monitoredOptions != nil && !instance.IgnoreMonitorAgent() {
```

Comment:
What if:

```yaml
pd_servers:
  - host: 172.16.5.138
    client_port: 2234
    peer_port: 2235
  - host: 172.16.5.138
    client_port: 2236
    peer_port: 2237
    ignore_exporter: true
```

Reply:
I've added a validator in `spec` to deny such configs.